--- expo.js.original	2024-11-10 09:08:19
+++ node_modules/react-native-edge-to-edge/dist/commonjs/expo.js	2024-11-10 09:08:23
@@ -19,10 +19,8 @@
     const {
       barStyle
     } = androidStatusBar;
+    const android = props?.android || {};
     const {
-      android = {}
-    } = props;
-    const {
       parentTheme = "Default"
     } = android;
     config.modResults.resources.style = config.modResults.resources.style?.map(style => {
\ No newline at end of file
